src/pages/recentDeliveries/RecentDeliveriesPage.tsx

```diff
@@ -184,10 +199,21 @@ const RecentDeliveriesObserverable = () => {
         </Col>
         <Col className={"text-start"}>{data.requestsConnection.totalCount} matching requests</Col>
 
-        <Col>
+        {/* <Col>
           <Button onClick={()=>{
-            jsdownload(data.requests,"blah.txt");
-          }}>Export</Button>
+            {CSVGenerate}
+            // jsdownload({CSVGenerate},"report.csv");
+          //  # jsdownload(data.requests,"blah.txt");
+          }}>Generate Report</Button>
+        </Col>
+      </Row> */}
+
+
+              <Col>
+          {/* <Button onClick={CSVGenerate}>Generate Report</Button> */}
+          <Button onClick={()=>{
```

Comment:
I think it's fine to still have the button callback function here even if it's still a work in progress. As long as it's not breaking the app while compiling

Reply:
keeping it as it is for now b/c Aaron and I are trying a couple of things here